bundles/org.openhab.binding.senseenergy/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="senseenergy" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>SenseEnergy Binding</name>
+	<description>This is the binding for Sense energy monitor (sense.com).</description>
+	<connection>hybrid</connection>
```

Comment:
Not sure if this makes sense (no pun intended), but would this sense monitor / box /device anounce itself on the local network? If so it woul dporbably use mDNS or similar protocol. If so it would be simple to add some xml for the binding suggestion finder.

Reply:
Unfortunately not that I can tell.  The user would still need to create the cloud-connection thing and enter their username/password though for the binding to work.